docs/content/preview/explore/ysql-language-features/going-beyond-sql/tablespaces.md

```diff
@@ -211,13 +211,17 @@ yugabyte=# SELECT * FROM multi_region_table;
 Time: 337.154 ms
 ```
 
-{{< tip title="Note" >}}
+## Leader preference
 
-The location of the leader can also play a role in the preceding latency, and the numbers can differ
-based on how far the leader is from the client node. However, controlling leader affinity
-is not supported via tablespaces yet. This feature is tracked [here](https://github.com/yugabyte/yugabyte-db/issues/8100).
+Having all tablet leaders reside closer to the user reduces the number of network hops, which reduces latency and increases performance. Leader preference allows you to specify the zones where you want the leaders to be placed when the system is stable and during outage of the preferred zones. You can specify non-zero contiguous integer values for each zone. When multiple zones have the same preference, the leaders will be evenly balanced between them.
```

Comment:
I think this section could do with far more detail. Please re-write this with more descriptions. First off, we would need definitions to what tablet leaders are, and how they can influence performance. If there is a relevant page in the docs explaining this already, add links to those. 
After you describe your example, explicitly say that all leaders will be present in us-east-1a zone. If all the nodes in us-east-1a zone fail, then the leaders will move to us-east-1b (because it is next in the hierarchy). It would also help to add more diagrams here: such as the Master UI page that shows the location of the tablet leaders.
Also thinking more, I would suggest adding a summary in https://docs.yugabyte.com/preview/explore/multi-region-deployments/row-level-geo-partitioning/ page (and link to this doc for more examples) as well.
Thirdly, note that all previous sections show the effect of tablespaces on latency (single zone has lesser latency than multi zone which is lesser than multi region). It would be good to show how the leader affects the latencies as well.